runtime/master/src/main/java/org/apache/nemo/runtime/master/scheduler/LambdaScheduler.java

```diff
@@ -0,0 +1,193 @@
+                     final ExecutorRegistry executorRegistry,
+                     final PlanStateManager planStateManager,
+                     final PipeManagerMaster pipeManagerMaster) {
+    this.planStateManager = planStateManager;
+    LOG.info("Using Lambda Scheduler");
+  }
+
+  @Override
+  public void schedulePlan(final PhysicalPlan submittedPhysicalPlan,
+                           final int maxScheduleAttempt) {
+
+    if (maxScheduleAttempt > 1) {
+      throw new UnsupportedOperationException();
+    }
+
+    planStateManager.updatePlan(submittedPhysicalPlan, maxScheduleAttempt);
+    planStateManager.storeJSON("submitted");
+
+    // We presume serverless function has been deployed and uploaded
+
+    // Prepare tasks
+    final List<Stage> allStages = submittedPhysicalPlan.getStageDAG().getTopologicalSort();
+    allStages.stream().forEach(stageToSchedule -> {
+      LOG.info("Execute new stage");
+      final List<StageEdge> stageIncomingEdges =
+        submittedPhysicalPlan.getStageDAG().getIncomingEdgesOf(stageToSchedule.getId());
+      final List<StageEdge> stageOutgoingEdges =
+        submittedPhysicalPlan.getStageDAG().getOutgoingEdgesOf(stageToSchedule.getId());
+
+      // the list of maps between vertex ID and readables.
+      final List<Map<String, Readable>> vertexIdToReadables = stageToSchedule.getVertexIdToReadables();
+      final List<String> taskIdsToSchedule = planStateManager.getTaskAttemptsToSchedule(stageToSchedule.getId());
+
+
+      taskIdsToSchedule.forEach(taskId -> {
+        AWSLambda client = AWSLambdaClientBuilder.standard().build();
+        final JavaConfigurationBuilder jcb = Tang.Factory.getTang().newConfigurationBuilder();
+        jcb.bindNamedParameter(TcpPortRangeBegin.class, "5000");
+        LOG.info("Schedule task" + taskId + "Init Conf builder");
+
+        final Injector injector = Tang.Factory.getTang().newInjector(jcb.build());
+        try {
+          final TcpPortProvider tcpPortProvider = injector.getInstance(TcpPortProvider.class);
+          final ChannelGroup serverChannelGroup = new DefaultChannelGroup(GlobalEventExecutor.INSTANCE);
+          LOG.info("Get tcp port");
+
+          //final OffloadingEventHandler nemoEventHandler = null;
+          // assigning event handler is not assigned so we pass null
+
+          NettyServerTransport nettyServerTransport =  new NettyServerTransport(
+            tcpPortProvider, new NettyServerSideChannelHandler(serverChannelGroup, null));;
+
+          LOG.info("Init netty server");
+          AWSLambdaAsync awsLambda = AWSLambdaAsyncClientBuilder.standard().withClientConfiguration(
+            new ClientConfiguration().withMaxConnections(500)).build();
+
+          LOG.info("Create request");
+          InvokeRequest request = new InvokeRequest()
+            // hard coded for now, need receiving user info
+            .withFunctionName("LambdaExecutor")
```

Comment:
why don't you create a static variable ? (e.g., `LAMBDA_FUNCTION_NAME` = "LambdaExecutor"`? )